ci/tlk.ps1

```diff
@@ -674,6 +676,7 @@ class TlkRecipe
                 } else {
                     throw ("Specify DGATEWAY_WEBCLIENT_PATH environment variable")
                 }
+                $DGatewayWebPlayer = "$($this.SourcePath)/webapp/player"
```

Comment:
disclaimer: I'm probably just not understanding (I'm only reading the PR without full context)

The player comes direct from the source tree (i.e. it's not something compiled)? 

Just reading this implies that there should be something in ./webapp/player, but I don't see anything in that path when I browse the repo?

Reply:
Probably a good catch. @irvingoujAtDevolution  modified that recently, maybe this branch wasn’t up to date?
- https://github.com/Devolutions/devolutions-gateway/pull/1169
- https://github.com/Devolutions/devolutions-gateway/pull/1163

Reply:
Excellent, thanks!